go/kbfs/libkbfs/prefetcher.go

```diff
@@ -428,7 +428,11 @@ func (p *blockPrefetcher) request(ctx context.Context, priority int,
 			pre.parents[ptr.RefNonce] = make(map[BlockPointer]<-chan struct{})
 		}
 		pre.parents[ptr.RefNonce][parentPtr] = parentPre.waitCh
-		p.log.CDebugf(ctx, "%d blocks to prefetch %t", pre.subtreeBlockCount, isParentNew)
+		if pre.subtreeBlockCount > 0 {
+			p.log.CDebugf(ctx,
+				"Prefetching %v, action=%s, numBlocks=%d, isPrefetchNew=%t",
```

Comment:
did you mean to put `isPrefetchNew` instead of `isParentNew`?

Reply:
I did indeed, thanks!